GitCommands/Git/GitModule.cs

```diff
@@ -27,6 +27,7 @@ namespace GitCommands
     [DebuggerDisplay("GitModule ( {" + nameof(WorkingDir) + "} )")]
     public sealed class GitModule : IGitModule
     {
+        public static readonly string GitError = "Git Error";
```

Comment:
I don't think it needs to be public or static.
```suggestion
        private const string GitError = "Git Error";
```

Reply:
OK (used in other modules initially, but creation of GitItemStatus should maybe be centralised anyway).